.github/workflows/CI.yml

```diff
@@ -41,7 +42,7 @@ jobs:
             ${{ runner.os }}-node-
 
       - name: Install Dependencies
-        run: npm ci
+        run: npm install
```

Comment:
Why was this change needed for this release?

Reply:
`npm ci` was breaking with this error `abbrev not accessible from nopt`  and I didn't have time to properly look into it as I need to start the applause cycle.

Reply:
Ah, ok. Interesting. Was just wondering about it since I remember you made tweaks to the workflow to allow the `npm ci` command to run.